spot_rl_experiments/spot_rl/utils/heuristic_nav.py

```diff
@@ -188,7 +188,7 @@ def push_forward_point_along_theta_by_offset(
     return (x, y)
 
 
-def get_me_arguments_for_image_search_fn(spot: Spot, angle):
+def get_arguments_for_image_search_fn(spot: Spot, angle):
```

Comment:
could you please remove `fn` in the function name here as we discussed yesterday? Thank you!